src/main/java/com/redhat/cloud/custompolicies/app/model/FactType.java

```diff
@@ -23,6 +23,7 @@
 
   STRING("STRING"),
   BOOLEAN("BOOLEAN"),
+  LIST("LIST"),
```

Comment:
Just a nitpick, but aren't these arrays actually (in JSON) ?

Reply:
For the moment this is meant as a hint to the UI what datatype a fact is, so that it can show the right set of comparators.
I can change it to array, but I don't think it really matters.